modules/ROOT/partials/configuration/importcss_file_filter.adoc

```diff
@@ -3,7 +3,7 @@
 
 This option enables you to add the CSS files that should be used for populating the styles drop down. This will go through any `+@import+` rules and only target the specified file. This option can be either a `+string+`, `+regexp+` or a `+function+`. This also allows you to import styles form existing files on the currently editable page in inline mode.
 
-Type: `+String+`
+Type: `+String+` or `+Regex+` or `+Function+`
```

Comment:
```suggestion
Type: `+String+`, `+Regex+`, or `+Function+`
```

Reply:
Like my previous question do we need examples for these now?

Reply:
Have added snippets for the missing regexp and function examples